src/common/benches/bitmap.rs

```diff
@@ -18,35 +18,39 @@ use risingwave_common::buffer::{Bitmap, BitmapIter};
 
 fn bench_bitmap(c: &mut Criterion) {
     const CHUNK_SIZE: usize = 1024;
-    let x = Bitmap::zeros(CHUNK_SIZE);
-    let y = Bitmap::ones(CHUNK_SIZE);
+    let bytes = vec![0x33; CHUNK_SIZE / 8];
```

Comment:
Nit: I prefer the bit notation here, since it's a bit map:
```suggestion
    let bytes = vec![0b00110011; CHUNK_SIZE / 8];
```